.github/workflows/local-node-test-reusable.yaml

```
@@ -0,0 +1,206 @@
+# SPDX-License-Identifier: Apache-2.0
+name: "Local Node E2E Tests Workflow"
+
+on:
+  workflow_dispatch:
+    inputs:
+      networkNodeTag:
+        description: "Specify desired Network Node image tag"
+        required: false
+        type: string
+        default: ""
+      mirrorNodeTag:
+        description: "Specify desired Mirror-Node image tag"
+        required: false
+        type: string
+        default: ""
+      relayTag:
+        description: "Specify desired Hedera JSON-RPC Relay tag"
+        required: false
+        type: string
+        default: ""
+  workflow_call:
+    inputs:
+      networkNodeTag:
+        description: "Specify desired Network Node image tag"
+        required: false
+        type: string
+        default: ""
+      mirrorNodeTag:
+        description: "Specify desired Mirror-Node image tag"
+        required: false
+        type: string
+        default: ""
+      relayTag:
+        description: "Specify desired Hedera JSON-RPC Relay tag"
+        required: false
+        type: string
+        default: ""
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  GRADLE_EXEC: "ionice -c 2 -n 2 nice -n 19 ./gradlew "
+  NPX_RUN: "ionice -c 2 -n 2 nice -n 19 npx "
+  NPM_RUN: "ionice -c 2 -n 2 nice -n 19 npm "
+
+permissions:
+  contents: read
+
+jobs:
+  local-node-test:
+    name: "Local Node Acceptance Tests"
+    timeout-minutes: 90
+    runs-on: hiero-block-node-linux-medium
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@4d991eb9b905ef189e4c376166672c3f2f230481 # v2.11.0
+        with:
+          egress-policy: audit
+
+      - name: Checkout Block Node
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          fetch-depth: 0
+
+      - name: Expand Shallow Clone for Spotless
+        run: |
+          if [ -f .git/shallow ]; then
+            git fetch --unshallow --no-recurse-submodules
+          else
+            echo "Repository is not shallow, no need to unshallow."
+          fi
+
+      - name: Set up JDK 21
+        uses: actions/setup-java@3a4f6e1af504cf6a31855fa899c6aa5355ba6c12 # v4.7.0
+        with:
+          distribution: "temurin"
+          java-version: "21.0.6"
+
+      - name: Cache Gradle packages
+        uses: actions/cache@d4323d4df104b026a6aa633fdb11d772146be0bf # v4.2.2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Build Block Node Docker Image
+        run: ${{ env.GRADLE_EXEC }} :block-node-server:createDockerImage
```

Should be possible to simplify these a little.
```suggestion
        run: ${GRADLE_EXEC} :block-node-server:createDockerImage
```

---

Fixed